Use all available style numbers by default, removing the ability to use the old
(deprecated) Style Byte Indicators. This is required by UDL, as it uses more
styles (56) than the default value provides (32).
Index: /as/komodo-devel/src/scintilla/src/Document.cxx
--- src/Document.cxx
+++ src/Document.cxx (unsaved)
@@ -90,8 +90,8 @@
 #endif
 	dbcsCodePage = 0;
 	lineEndBitSet = SC_LINE_END_TYPE_DEFAULT;
-	stylingBits = 5;
-	stylingBitsMask = 0x1F;
+	stylingBits = 8;
+	stylingBitsMask = 0xFF;
 	stylingMask = 0;
 	endStyled = 0;
 	styleClock = 0;
